obs-websocket-dotnet-tests/UnitTest_Types.cs

```diff
@@ -40,8 +40,8 @@ public void OBSScene_BuildFromJSON()
             var scene = new OBSScene(data);
 
             Assert.AreEqual(sceneName, scene.Name);
-            Assert.AreEqual(1, scene.Items.Count);
-            Assert.AreEqual(itemName, scene.Items[0].SourceName);
+            //Assert.AreEqual(1, scene.Items.Count);
```

Comment:
Missing // TODO
Also, please state what "to do" // TODO Uncomment towards v5 release